Source/Project64/UserInterface/SettingsConfig.cpp

```diff
@@ -251,9 +251,9 @@ void CSettingConfig::ApplySettings(bool UpdateScreen)
     if (!GameIni.empty())
     {
         stdstr GoodName;
-        if (!g_Settings->LoadStringVal(Game_GoodName, GoodName))
+        if (!g_Settings->LoadStringVal(Cfg_GoodName, GoodName))
```

Comment:
Ok, changed it to read from Rdb, write to Cfg.  I should have done what I thought in the first place.

Good to go?